src/wallet/message_store.h

```diff
@@ -202,7 +203,8 @@ namespace mms
   class message_store
   {
   public:
-    message_store();
+    message_store(std::unique_ptr<epee::net_utils::http::abstract_http_client> http_client = std::unique_ptr<epee::net_utils::http::http_simple_client>(new epee::net_utils::http::http_simple_client()));
```

Comment:
Why do we need a default `= std::unique_ptr<epee::net_utils::http::http_simple_client>(new epee::net_utils::http::http_simple_client())`?